### PR TITLE
 introduce custom domain name with default value to cf-app.com.

### PR DESCRIPTION
**Pull request by o-orand**
Ignore DOMAIN_NAME in credential filter.
Use domain name only in bosh-lite task.
Use fallback default value in scripts when not set by tasks.

### CODE REVIEW

**Comment by RochesterinNYC** [comment]
@o-orand thanks for the pull request. Generally looks great.

I notice that you flip between specifying `cf-apps` and `cf-app` for the domain in some places. Could you fix that please?

File: bin/update-all-the-pipelines [hunk 1]
```diff
@@ -5,6 +5,22 @@ require 'yaml'
...
+
+lpass_config= {}
+lpass_yaml_data=%x{lpass show #{LPASS_CONCOURSE_PRIVATE} --notes && lpass show #{LPASS_DEPLOYMENTS_BUILDPACKS} --notes}
+if $?.exitstatus != 0
+  puts "WARNING: ignoring lastpass config file. An error occured while processing #{LPASS_CONCOURSE_PRIVATE} and #{LPASS_DEPLOYMENTS_BUILDPACKS}"
+else
+  lpass_config = YAML.load(lpass_yaml_data)
+end
+
+full_config = public_config.merge(lpass_config)
```
Comment (RochesterinNYC) [inline bin/update-all-the-pipelines]: What is the purpose of this `full_config` variable and the merge? It looks like only `domain-name` is only fetched from this `full_config`, and `domain-name` is specified in the public config, so I don't see why the `public_config` needs to be merged with the `lpass_config`?

File: public-config.yml [hunk 2]
```diff
@@ -1,3 +1,4 @@
 # This file holds public credentials (like github uri, github organization) to ease creation of a new pipeline instance.
 buildpacks-github-org: cloudfoundry
 buildpacks-binaries-s3-bucket: pivotal-buildpacks
+domain-name: cf-apps.com
```
Comment (RochesterinNYC) [inline public-config.yml]: I think this should be `cf-app.com`?

File: bin/update-all-the-pipelines (hunk 1 above)
Comment (o-orand) [inline bin/update-all-the-pipelines]: It possible to override value define in `public_config` with some defined in `lpass-config`. It's the same behavior already in place in [set-pipeline](https://github.com/cloudfoundry/buildpacks-ci/blob/master/bin/update-all-the-pipelines#L22-L23). Is it clearer ?

File: public-config.yml (hunk 2 above)
Comment (o-orand) [inline public-config.yml]: You're right !

**Comment by o-orand** [comment]
@RochesterinNYC typo fixed. Thanks !

**Comment by RochesterinNYC** [comment]
Thanks.

**Pull request merged**